src/File/Upload.php

```diff
@@ -109,7 +110,7 @@ public function getFiles($file = null)
      */
     public function setFiles($files = [])
     {
-        if (count($files) === 0) {
+        if ((is_array($files) || $files instanceof Countable) && count($files) === 0) {
```

Comment:
Well, it seems I was wrong: https://3v4l.org/Efkoc
So I guess a `$_FILES` test case is missing; condition changed.